proposals/0000-type-level-literals.rst

```diff
@@ -0,0 +1,92 @@
+Separate tag for type-level literals
```

Comment:
```suggestion
Separate extension for type-level literals
```
I've never seen anyone call extensions "tags" before. "Tag" also makes me think of the tags of data constructors, which has nothing to do with what this is about.

Reply:
fixed, thanks